templates/rss.ts

```diff
@@ -0,0 +1,33 @@
+export const rssTemplate = `<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
wouldn't it make sense for this file to be `tsx` ?

Reply:
@moshmage as this file only exports a string and not HTML, i think we can use the `.ts` extension

Reply:
if this is a template for handlebars, use `hbs` instead; If this is "html with javascript" use `tsx`